cypress/integration/backend-admin-ui/test_2_dataset.js

```diff
@@ -6,82 +6,110 @@ context("Admin UI Single Dataset", () => {
-        before(() => { visitAdminUI(`datasets/${testDSID}`); });
+      before(() => { visitAdminUI(`datasets/${testDSID}`); });
 
-        it("Can see the datasets page", () => {
-            cy.contains(`Dataset ${testDSLabel}`);
-        });
+      it("Can see the datasets page", () => {
+        cy.contains(`Dataset ${testDSLabel}`);
+      });
 
-        it("Can change the label", () => {
-            cy.get('[data-test-id="editableText-btn"]').click();
-            cy.get('[data-test-id="editableText-input"]').clear().type(`NEW ${testDSLabel}`);
-            cy.get('[data-test-id="editableText-form"]').submit();
+      it("Can change the label", () => {
+        cy.get('[data-test-id="editableText-btn"]').click();
+        cy.get('[data-test-id="editableText-input"]').clear().type(`NEW ${testDSLabel}`);
+        cy.get('[data-test-id="editableText-form"]').submit();
 
-            cy.contains(`Dataset NEW ${testDSLabel}`);
-        });
+        cy.contains(`Dataset NEW ${testDSLabel}`);
+      });
     });
 
     describe("Can upload test table and concept", () => {
-        before(() => { visitAdminUI(`datasets/${testDSID}`); });
-
-        it("Can upload test table", () => {
-            cy.get('[data-test-id="upload-select"]').select('Table JSON');
-            cy.get('[data-test-id="upload-input"]').selectFile('./cypress/support/test_data/all_types.table.json');
-            cy.get('[data-test-id="upload-btn"]').click();
-            cy.wait(3000).reload();
-        });
-
-        it("Is new table visible", () => {
-            cy.get('[data-test-id="accordion-Tables"]').contains('td', `${testDSID}.table`);
-        });
-
-        it("Can upload concept table", () => {
-            cy.get('[data-test-id="upload-select"]').select('Concept JSON');
-            cy.get('[data-test-id="upload-input"]').selectFile('./cypress/support/test_data/all_types.concept.json');
-            cy.get('[data-test-id="upload-btn"]').click();
-            cy.wait(3000).reload();
-        });
-
-        it("Is new concept visible", () => {
-            cy.get('[data-test-id="accordion-Concepts"]').contains('td', `${testDSID}.concept1`);
-        });
+      beforeEach(() => { visitAdminUI(`datasets/${testDSID}`); });
+
+      it("Can upload test table", () => {
+        cy.get('[data-test-id="upload-select"]').select('Table JSON');
+        cy.get('[data-test-id="upload-input"]').selectFile('./cypress/support/test_data/all_types.table.json');
+        cy.get('[data-test-id="upload-btn"]').click();
+      });
+
+      it("Is new table visible", () => {
+        cy.get('[data-test-id="accordion-Tables"]').contains('td', `${testDSID}.table`);
+      });
+
+      it("Can upload concept table", () => {
+        cy.get('[data-test-id="upload-select"]').select('Concept JSON');
+        cy.get('[data-test-id="upload-input"]').selectFile('./cypress/support/test_data/all_types.concept.json');
+        cy.get('[data-test-id="upload-btn"]').click();
```

Comment:
Hier müsste noch gecheckt werden, dass das Upload des Concept in der UI etwas geändert hat